docs/05-reference/api/update-asset.mdx

```diff
@@ -140,3 +140,29 @@ curl -X PATCH https://livepeer.studio/api/asset/{id} \
 
 Returns a `200 OK` response with an object containing the updated properties
 indicates that the `asset` was successfully updated.
+
+## Update Access Control Policy
```

Comment:
We should not give this example for assets, as assets are exactly what we **dont** support. This example must be in the `update-stream` documentation instead. Only when we add support for that on assets that we should document it.